<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="Label_Username">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Password">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Input_Username">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Input_Password">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Login">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QListView" name="Image_List">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>256</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>10</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Token</string>
    </property>
   </widget>
   <widget class="QLabel" name="Token_Value">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>10</y>
      <width>59</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AddImage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DeleteImage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DownloadImage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Download Image</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Input_Add_Name">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>370</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Image Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Input_Delete_Name">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>400</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Image Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Input_Download_Name">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>430</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Image Name</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="Viewer">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>100</y>
      <width>401</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>List of Images</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Click_View_List">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>320</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>View Image List</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Display_Button">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>70</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Display This Image</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ImageName_to_Display">
    <property name="geometry">
     <rect>
      <x>482</x>
      <y>70</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Login_Color">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Label_IP">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>IP Address</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Port">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>40</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Port Number</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Input_IP">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Input_Port">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>40</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Insert">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>30</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Insert</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
